app/scripts/metamask-controller.js

```diff
@@ -2578,10 +2576,7 @@ export default class MetamaskController extends EventEmitter {
       throw new Error('Primary keyring mnemonic unavailable.');
     }
 
-    const recoveredIndices = Array.from(
-      new Uint16Array(new Uint8Array(keyring.mnemonic).buffer),
-    );
-    return recoveredIndices.map((i) => englishWordlist[i]).join(' ');
+    return keyring.mnemonic;
```

Comment:
Did you check if this function is used anywhere else, which may not accept `Uint8Array` mnemonic phrases?

Reply:
Yeah. The only reference to the function is exposing it as a hook to snap RPC methods.